MAUI/TreeMap/right-to-left.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Drilldown in .NET MAUI TreeMap Control | Syncfusion
+description: Learn here all about the flow direction feature of Syncfusion .NET MAUI TreeMap (SfTreeMap) control.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Right To Left in .NET MAUI TreeMap (SfTreeMap)
+
+SfTreeMap supports changing the flow to the right-to-left direction by setting the FlowDirection to RightToLeft. TreeMap supports RTL when the flow direction of the device is changed.
```

Comment:
SfTreeMap supports changing the flow to the right-to-left direction by setting the FlowDirection to RightToLeft.

Reply:
Updated the changes na